docs/stale-subvolume.md

```diff
@@ -0,0 +1,38 @@
+# subvolume cleanup
+
+The subvolume command is used to clean the stale subvolume/images
+which has no parent-pvc attached to it.
+The command would list out all such subvolume/images which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+Few cases are mentioned below:
```

Comment:
Why do these cases cause stale subvolumes? If there is a description of stale subvolumes already in another doc such as ceph docs, you could also link it instead of describing all the scenarios.

Reply:
added the explanation in above comment. that may have these corner cases where we can have stale subvolumes.

Reply:
The previous paragraph sounds good. How about if we just remove this list (lines 9-13)? Seems like we don't need to list all these scenarios.